exercises/isogram/cases_test.go

```diff
@@ -54,4 +54,9 @@ var testCases = []struct {
 		input:       "accentor",
 		expected:    false,
 	},
+	{
+		description: "longest German isogram",
+		input:       "Heizölrückstoßabdämpfung",
+		expected:    true,
+	},
```

Comment:
Nice test case.  The cases_test.go file is generated by [isogram/.meta/gen.go](https://github.com/exercism/go/blob/master/exercises/isogram/.meta/gen.go) from [the canonical-data.json file](https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/canonical-data.json) so I'm thinking it may be best to suggest the change in [that repository](https://github.com/exercism/problem-specifications).  If UTF-8 is permitted for the other language tracks it may be acceptable. If not, we could add another test case in  isogram_test.go.

Reply:
ah thanks. ill look at that file and check out on eof the other languages to see